restcomm/restcomm.dao/src/main/java/org/mobicents/servlet/restcomm/dao/DaoUtils.java

```diff
@@ -95,6 +97,14 @@ public static Integer readInteger(final Object object) {
         }
     }
 
+    public static BigInteger readBigInteger(final Object object) {
+        if (object != null) {
+            return (BigInteger) object;
```

Comment:
not sure what is added value here,apart from isolating the casting... this may be designed differently, i will keep an eye here